OpenSim/Common/TableSource.h

```diff
@@ -65,6 +65,13 @@ class TableSource_ : public Component {
     OpenSim_DECLARE_CONCRETE_OBJECT_T(TableSource_, ET, Component);
 
 public:
+    OpenSim_DECLARE_PROPERTY(filename, std::string,
+                             "Path to the file to populate the TableSource_ "
+                             "with.");
+    OpenSim_DECLARE_PROPERTY(tablename, std::string,
+                             "Name of the table in the file to populate the "
+                             "TableSource_ with. Ex. 'markers', 'forces'.");
```

Comment:
Can you clarify that this is necessary only for certain file types (here and elsewhere)?